app/helpers/datasets_helper.rb

```diff
@@ -29,6 +29,11 @@ def displayed_date(dataset)
     end
   end
 
+  def to_markdown(content)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: false)
```

Comment:
@rossjones did you look at using https://kramdown.gettalong.org/rdoc/ – it's the library used by https://github.com/alphagov/govspeak

Reply:
Ah no, I just used redcarpet through familiarity (I've used it before).
We probably should switch for consistency

Reply:
Another option maybe would be `Govspeak::Document.new(content).to_sanitized_html`

Reply:
@rossjones yes no probs

@andrewgarner ta